docs/quickstart.rst

```diff
@@ -0,0 +1,69 @@
+.. _quickstart:
```

Comment:
this looks like it should be combined with installation? it looks like it was in mms (there's a reference to compose file from mms), or maybe just a quick start for setting dependencies since installation goes into setting up mms